File: plugins/de.ovgu.featureide.examples/featureide_examples/GPL-FH-Java/features/Base/GPL/EdgeIfc.java
//created on: Sun Jul 13 23:04:15 CDT 2003

package GPL;

public interface EdgeIfc
{
    public Vertex getStart( );
    public Vertex getEnd( );
    public void display( );


    public void setWeight( int weight );
 //   public int getWeight();

    public Vertex getOtherVertex( Vertex vertex );

    public void adjustAdorns( EdgeIfc the_edge );
}

File: plugins/de.ovgu.featureide.examples/featureide_examples/GPL-FH-CSharp/features/Base/EdgeIfc.cs
/*
 * //created on: Sun Jul 13 23:04:15 CDT 2003

layer base;
*/
public interface EdgeIfc
{
    Vertex GetStart( );
    Vertex GetEnd( );
    void display( );


    void setWeight( int weight );
 //   public int GetWeight();

    Vertex GetOtherVertex( Vertex vertex );

    void adjustAdorns( EdgeIfc the_edge );
}